OpenRA.Mods.Common/Traits/World/WeatherOverlay.cs

```diff
@@ -0,0 +1,421 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using OpenRA.Graphics;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Adds an overlay for weather animations based on dropping particles (snowfalkes, raindrops, sand grains)")]
+	public class WeatherOverlayInfo : ITraitInfo
+	{
+		[Desc("On-off switch for the weather overlay")]
+		public readonly bool WeatherEnabled = true;
```

Comment:
It was ment for some lua scripters who like to enable it, after some time has passed.
I don't get the differences of a bool or an Enabled property in that case.